src/components/Button/Button.js

```diff
@@ -176,8 +176,8 @@ const getLabelTopMargin = (props) => {
   return getButtonFontSize(props) * 0.18;
 };
 
-const getButtonOpacity = ({ disabled, isLoading, theme }) => {
-  if (disabled && !isLoading) {
+const getButtonOpacity = ({ disabled, theme }) => {
```

Comment:
This doesn't belong to the scope, but it confuses every time when submit button is loading and is disabled it's still showing regular opacity. Let me know if you have concerns.